contracts/test/utils/Fixtures.sol

```diff
@@ -16,6 +17,8 @@ struct FeiTestAddresses {
     address minterAddress;
     address burnerAddress;
     address guardianAddress;
+    address pcvGuardianAdminAddress;
```

Comment:
Not down to you, but I don't see why we need to append the string `Address` onto these names. Probably cleaner to just do `address minter`, `address pcvGuardianAdmin` etc